ovos_plugin_manager/templates/media.py

```diff
@@ -4,7 +4,10 @@
 from ovos_utils.log import LOG
 from ovos_utils.messagebus import FakeBus
 
-from ovos_utils.ocp import MediaState, PlayerState, TrackState
+try:
+    from ovos_utils.ocp import MediaState, PlayerState, TrackState
+except ImportError:
+    MediaState, PlayerState, TrackState = None, None, None
```

Comment:
this module didnt even exist before utils 0.1.0, nothing supporting utils 0.0.X should be importing from here at all?

feels like this is just hiding the error / kicking it down to plugin load time?

if this file is imported shouldnt it fail on import if utils 0.1.0 is not present?

Reply:
This is imported internally, so without this change the exception is always raised; this defers the exception until the import is used

Reply:
maybe we should revert the audio.py remplate then, and mark it completely for removal  (as its intended to be replaced by ovos-media plugins and fully removed)

Reply:
ie, partially revert https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/207

keeping the AudioService mostly unchanged

currently old plugins are not used at all so this is safe for new code, and keeps what old code expected around  (the handful of deprecated audio plugins)